### ISSUE TITLE
Hybrid Android 6.0 Litmus Failing

### ISSUE THREAD

**Issue by mischiefgithub** [emailmonday/Cerberus#224]
Looking to use Cerberus as base for starting emails.  Notice that the Hybrid is failing on Android 6.0.  Not sure if it used to work and now is cutting on the HTML.

![android6-vertical-allowed-1366](https://user-images.githubusercontent.com/3453254/59851478-bd9bc780-933a-11e9-9dfd-dc91feb670bc.png)

I just pasted the RAW code into Litmus Builder and ran previews.

**Comment by TedGoas** [comment]
@mischiefgithub What part(s) is failing?

**Comment by mischiefgithub** [comment]
The bottom section of the page is not being rendered.  It seems to be stopping at the lower section with an image on one side and text with button on the other.

**Comment by TedGoas** [comment]
@mischiefgithub Aye, I see this in Litmus for Android 5.1 and Android 6.0. I feel like I've heard about this bug before and I think it's an issue with Litmus's preview rather than these templates. I think the issue is that **Litmus stops capturing previews after a certain height for these clients.**

Here are few things I tried that lead me to believe that's what's happening:

1. All of the other other Litmus previews on Android clients work (Android 4.0, all Gmail's, Outlook).
2. Next I tried testing the `cerberus-fluid.html` templates (which is smaller), and Litmus displayed the full preview in Android 5.1 and Android 6.0.
3. Lastly I removed a few sections from the top of `cerberus-hybrid.html`, and Litmus displayed the full preview in Android 5.1 and Android 6.0 (down to the bottom).

I'm not sure _why_ these two clients, but it seems like a bug in Litmus.
